### PR TITLE
Updated DFU tutorial, minor quest fixes

### PR DESCRIPTION
**Pull request by JayH2971**
Warning: I don't know what "multiple PRs" look like. I am a github novice.

This contains very basic quest fixes, almost unnoticeable. It also has my updated DFU tutorial in file _TUTOR__.txt.

### CODE REVIEW

File: Assets/StreamingAssets/Quests/A0C01Y01.txt
Comment (KABoissonneault) [inline Assets/StreamingAssets/Quests/A0C01Y01.txt:122]: When you change this from 23:59 to 22:59, you added the `_latehour_` task below.
https://github.com/Interkarma/daggerfall-unity/commit/21fadbd6cbf32888bb070fcbe786bd5e4f87f8db

Is the `_latehour_` still needed or not?

File: Assets/StreamingAssets/Quests/M0B00Y06.txt
Comment (KABoissonneault) [inline Assets/StreamingAssets/Quests/M0B00Y06.txt:46]: Missing a T at the start.

**Comment by KABoissonneault** [comment]
Are the <ce> tokens on rumors an issue?

From what I'm seeing so far, it seems quests generally tend to not have <ce> on Rumors, but generally tend to have it on Questor. This makes sense to me, since the rumors is in the Talk UI, while the Questor is a popup.
Your changes seems to remove them on both Rumors and Questor, though. Was it intended to remove them from Questor? Also, should we do a pass and remove <ce> from all rumors?

File: Assets/StreamingAssets/Quests/A0C01Y01.txt
Comment (JayH2971) [reply Assets/StreamingAssets/Quests/A0C01Y01.txt]: Good eye. I added latehour to ensure the enemy would have to spawn somehow, and that happens at 11PM. I restored the original spawning window and kept latehour as it is.

File: Assets/StreamingAssets/Quests/_TUTOR__.txt
Comment (KABoissonneault) [inline Assets/StreamingAssets/Quests/_TUTOR__.txt:12]: You got a few double spaces in there. I'd do a pass with Find "  " and replace it with " "

**Comment by JayH2971** [comment]
> This makes sense to me, since the rumors is in the Talk UI, while the Questor is a popup.
The questor only gives a popup message if it's invoked in a quest action like "say" or "givepcreward." The messages  "QuestorPostfailure" and "QuestorPostsuccess" only appear if you enter the conversation window with the questgiver after the quest ends. A lot of them are misformatted, and I haven't fixed them.

For the other questions, I need some more clarification on them.

**Comment by KABoissonneault** [comment]
After discussing offline, it seems any "Talk UI" message on multiple lines does not get displayed properly. This is an issue with many quests in the game. We might wanna just fix them all

**Comment by Jagget** [comment]
Are you saying that RumorsDuringQuest, RumorsPostfailure, RumorsPostsuccess, QuestorPostsuccess, and QuestorPostfailure only ever displayed in dialog interface, and that all 5 sections do not need formatting? 

Btw, there are `say QuestorPostsuccess` in S0000003, and `say QuestorPostfailure` in S0000013

**Comment by JayH2971** [comment]
Yes, those five fields only are used in the Talk UI. Kab tested it earlier today, and <ce> lines and multiple-line formatting do not interfere with their appearance inside the Talk UI; the message is displayed fine.

Finding Questor messages in the main quest is odd but that's classic Daggerfall data. No doubt one of many things they changed during development.

**Review by Jagget — APPROVED** [review]

**Comment by KABoissonneault** [comment]
Yes I want to be on record that after testing offline, the Talk UI does handle this, by simply replacing <ce> tokens with spaces. Gonna run the PR locally to see what the tutorial looks like in action now before approving

**Comment by ajrb** [comment]
I've read this thread a couple of times and I am still confused. Are formatting tags like <ce> an issue for rumors / postsuccess etc message that are used in talk window really an issue? I'd like to know because I have 2 blocks like that in my R&R quest.

**Comment by KABoissonneault** [comment]
@ajrb In my tests, no. When I initially reported this, this was based on my conversation with JayH, but after that conversation, I was not able to reproduce the issue. We're moving forward as if it's not an issue. However, I still think it's better if new quests avoid using <ce> on Talk text, simply before it's not used and can lead to confusion when reading the quest

**Comment by KABoissonneault** [comment]
I _think_ I'd like to prioritize this for 1.1. The earlier we release this, the more feedback we can get. However, I think this could be a controversial change, if it were to be less useful than the original. Maybe some purists won't like it, but this can probably be modded back in easily.

I'll test it myself, and maybe we could playtest it on Lysandus' Tomb?

**Comment by KABoissonneault** [comment]
Some feedback on the Tutorial quest

image

- The swing type can be changed anytime in Controls -> Advanced. I'd remove mention of the title screen, to avoid having people restart the game to change swing type. Just mention that after Controls, it's under  Advanced
- Is there supposed to be a Yes No?

image

- Along with the first one, I find 5 seconds for a call to action might be a bit short. I'm worried someone will start looking at their keyboard for too long, and then the next tutorial will start. It's not like the user is stuck waiting, I think it's fine to have slightly longer waits. 15 seconds on could be fine. I guess we could playtest it and adjust.

image

- Not sure about the phrasing on the saving throws. Yes, some enemies have less than 50% on their saving throws, but I don't want to make it seem like magic is only meant to be used against them. Having half your spells miss is how magic is expected to work. Maybe just mention that it's normal for half of your spells to miss.

image
- Not sure about the suggestion of Daggerfall. Gothway Garden is an iconic entry town. Pretty arbitrary I suppose

image
- Knightly Orders are also unique, might be worth mentioning

image
- The tutorial avoids mentioning hotkeys for the most part. I kind of get why you do it here, but worth considering that they might have been changed

image
- Yay

Overall, I'm kind of okay with the short timers on messages. I get it might be better to just get them over with. Just one thing, is that I find it pretty difficult to get the RumorsDuringQuest messages with your version. Do you think we could replace them with RumorsPostSuccess? You'd get helper tips for 7 days after reading all of it. You still get the basic plot hooks if you said No to the quest (assuming you make it to a city in 7 days).

**Comment by JayH2971** [comment]
Okay, sounds good. Most of those are small changes, not hard to do. The reason I chose Daggerfall City is because it's the only place with a permanent location in the region of Daggerfall, so I thought I could do a "when enters" condition on it, but even that failed, so I changed it to "any city." I'll have to look up what location Gothway Garden is.

**Comment by KABoissonneault** [comment]
Information I like from the original that I see missing here.

Object activation and loot piles
image
- At least the part about activating the door and telling users to check corpses for loot
- Page 9 does tell you about activating the throne later on, but I feel the first door and basic loot piles are essential to mention early

Inventory window
image
- A bit more awkward since messages go by so far, you'd get this prompt before you actually found it. Maybe it's fine

> The reason I chose Daggerfall City is because it's the only place with a permanent location in the region of Daggerfall

Yeah to be fair so did the original. My bad, it's fine like this
image

**Comment by JayH2971** [comment]
Good points, I'm fine with them. I'll try to find a chance to change them.

File: Assets/StreamingAssets/Quests/_TUTOR__.txt
Comment (KABoissonneault) [inline Assets/StreamingAssets/Quests/_TUTOR__.txt:12]: Do these only occur if you say No to the tutorial, or does completing the tutorial to the end count as failure? I guess I'll test it out

Comment (KABoissonneault) [reply Assets/StreamingAssets/Quests/_TUTOR__.txt:12]: Oh just saw that the last page just calls the _no_ task, got it

Comment (JayH2971) [reply Assets/StreamingAssets/Quests/_TUTOR__.txt:12]: The way this is written, all ends lead to failure. The only way to get a quest to be a success is if you do "give pc ITEM" of some kind, or "give pc nothing." They invoke message 1004 as well. Anything without them is a failure.

**Review by KABoissonneault — APPROVED** [review]

**Review by KABoissonneault — CHANGES_REQUESTED** [review]
I think you should do a pass with SDF Fonts turned off to make sure the maximum width is similar to the original
image

**Comment by JayH2971** [comment]
It's too bad I'm only learning about this now. A lot of QP1 would probably have the same problem. But for this I think it's fixed, I tested every window in-game on non-SDF now.

**Comment by KABoissonneault** [comment]
Much better. I still wish the message boxes would fit inside retro frames, but maybe we need a tool to detect and message sizes in quests instead. We'll go with this for now

image
image

**Review by KABoissonneault — APPROVED** [review]

**Review by KABoissonneault — APPROVED** [review]

**Comment by Mentill** [comment]
I don't know how much additional work was going to be put into this, but as an FYI I've proofread all of the quest files and intend to do a PR for them if/when my current PR is approved.
I'll work my changes around these, however that'll be easier if fewer quest files have typos fixed here.

**Comment by JayH2971** [comment]
I don't intend to add anything else to this PR, unless someone finds a game-breaking bug of some kind. It's easier on Kab to let this one settle, so it doesn't have to get approved again.

I'll defer to the other maintainers on the question of text fixes.

**Review by petchema — APPROVED** [review]

**Pull request merged**
